src/components/Pricing.tsx

```diff
@@ -3,20 +3,96 @@ import { motion } from 'framer-motion';
 import { config } from '../config';
 import { CheckArrowIcon } from '../assets/icons/CheckArrowIcon';
 
-const pricing = [
+const pricing: Pricing[] = [
   {
     title: 'Open Beta',
     description: 'For early adopters with a sense of adventure.',
     price: 20,
     features: [
-      'Unlimited apps',
-      'Unlimited updates',
+      '10 apps',
+      'Up to 1,000,000 users',
       'Access to private Discord',
       'Live support',
     ],
+    cta: {
+      title: 'Get Started',
+      link: config.docsUrl,
+    },
+  },
+  {
+    title: 'Enterprise',
```

Comment:
Kept this in since I already had it but happy to remove if we don't want it. I personally don't think it hurts 🤷